docs/UsersGuide.asciidoc

```diff
@@ -271,6 +271,55 @@ provide a rename target, for example
 ISO_1_DECOMPRESS_URL=http://host/foo2.iso.xz and ISO_1=foo.iso
 
 
+== Asset cleanup ==
+
+Since assets consume a huge amount of disk space, openQA removes assets
+automatically according to configurable limits.
+
+=== Cleanup stragegy
+
+openQA frequently checks whether assets need to be removed according to
+the configured limit.
+
+To determine which assets can be removed, it looks by which job groups
+a certain asset is used. If at least one job within a certain job group
+is using an asset, the asset is considered to be used by that group.
+
+So an asset can be accounted to multiple groups. The assets table which is
+accessible via the admin menu shows these groups for each asset and also
+the latest job.
+
+If the size limit for assets by a certain group is exceeded, openQA will
+delete those assets:
+
+* Assets belonging to old jobs a preferred.
+* Assets belonging to jobs which are still scheduled or running will not be
+  considered at all.
+* Assets which also belong to another group which still has space left are
+  not considered as well.
+
+Assets which don't belong to any group are deleted after a configurable
+duration (by default 14 days).
+
+=== Configure limit for assets within groups
+
+To configure the maximum size for assets per group, open 'Job groups' in
+the operators menu and select a group. The size limit for assets can be
+configured under 'Edit job group properties'. It also shows the size of
+assets which belong to that group and not to any other group.
+
+Job groups inherit the size limit from their parent group unless the limit
+is set explicitely. The default size limit for groups can be adjusted in
+the +default_group_limits+ section of the openQA config file.
+
+=== Configure limit for groupless assets
+
+Assets only belonging to jobs not within a group are deleted automatically
+after 14 days. That duration can be adjusted by setting
```

Comment:
I would be a bit more careful by restarting the default values, you know, code changes, documentation becomes outdated …